### PR TITLE
nxos_ospf_vrf: Add 'bfd' support

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
##### SUMMARY
Add support for `bfd` state in `nxos_ospf_vrf`

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
`nxos_ospf_vrf`

##### ADDITIONAL INFORMATION
```paste below

```

### CODE REVIEW

**Comment by mikewiebe** [comment]
LGTM

File: lib/ansible/modules/network/nxos/nxos_ospf_vrf.py [hunk 1]
```diff
@@ -90,6 +90,12 @@
     description:
       - Specifies the reference bandwidth used to assign OSPF cost.
         Valid values are an integer, in Mbps, or the keyword 'default'.
+  bfd:
+    description:
+      - Enables BFD on all OSPF interfaces.
+      - "Dependency: 'feature bfd'"
+    version_added: "2.9"
+    type: bool
```
Comment (Qalthos) [inline lib/ansible/modules/network/nxos/nxos_ospf_vrf.py]: You've listed a default in the argspec, but not in the documentation
```suggestion
    type: bool
    default: false
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/nxos/nxos_ospf_vrf.py (hunk 1 above)
Comment (Qalthos) [inline lib/ansible/modules/network/nxos/nxos_ospf_vrf.py]: You've listed a default in the argspec, but not in the documentation
```suggestion
    type: bool
    default: false
```

**Comment by chrisvanheuveln** [comment]
I did a rebase and push --force but... *something* happened and it wiped out my earlier commits, which apparently then caused it to auto-close this PR. I wasn't sure if the branch was fixable so I just created a new branch and patched the commits to it. The replacement PR is #57425

**Pull request closed without merging**